docs/section4/4-6.md

```diff
@@ -0,0 +1,30 @@
+# Airside Systems – Rule 4-6
+
+**Rule ID:** 4-6
+**Rule Description:** HVAC fans in the proposed design model shall remain on during unoccupied hours in systems primarily serving computer rooms.  
+**Rule Assertion:** P-RMR = expected value                                           
+**Appendix G Section:** Section G3.1-4 Schedules exception #3 for the proposed building  
+**Appendix G Section Reference:** None  
+**Data Lookup:** None
+
+**Evaluation Context:** Evaluate each HeatingVentilationAirAconditioningSystem 
+**Applicability Checks:** 
+
+1. Applies to hvac systems primarily serving computer rooms.
+
+**Function Call:** 
+
+1. get_HVAC_systems_primarily_serving_comp_rooms()
+
+**Manual Check:** None  
+ 
+**Rule Logic:**  
+**Applicability Check 1:** 
+- Get list of hvac systems that primary serve computer rooms: `hvac_systems_primarily_serving_comp_rooms_list_p = get_HVAC_systems_primarily_serving_comp_rooms (P_RMR)`
+- For each hvac_p in hvac_systems_primarily_serving_comp_rooms_list_p: `for hvac_p in hvac_systems_primarily_serving_comp_rooms_list_p:`
+    - Get operation during unoccupied periods: `operation_during_unoccupied_p = hvac_p.fan_systems[0].operation_during_unoccupied`  
+    **Rule Assertion:**
+    - Case 1: For each applicable hvac system, if fan system operation during unoccupied periods is continuous in the P_RMR: `if operation_during_unoccupied_p == “CONTINUOUS”: PASS`
+    - Case 2: Else, for each applicable hvac system system: `Else: UNDETERMINED and raise_message "THE HVAC SYSTEM SERVES ZONE(S) THAT INCLUDE COMPUTER ROOMS AND THEREFORE MAY WARRANT CONTINUOUS OPERATION DURING UNOCCUPIED HOURS PER SECTION G3.1-4 SCHEDULES EXCEPTION #3 FOR THE PROPOSED BUILDING. VERIFY WITH A MANUAL CHECK." `
+
```

Comment:
Question for @mkarpman - should case 2 be fail and not 'undetermined'?

Reply:
Yes, I agree that it should be fail for consistency with 4-5.

Reply:
Done